### PR TITLE
added blurb about sensor locations to the about page

### LINKED ISSUE #639: Add explanation of sensor locations
People were confused why they couldn't see the AQI everywhere in the city. Add clarification to the About page that explains that there are a limited number of sensors located in people's yards, etc.

### PR DESCRIPTION
**Pull request by akampen**
Closes #639

### CODE REVIEW

**Comment by akampen** [comment]
I also re-ordered the jump-links so that they appear in the same order as the content (and ensure that they are in two columns, not 3, which was overlooked in the PurpleAir-blurb PR)

File: public/locales/en/about.json [hunk 1]
```diff
@@ -32,6 +32,10 @@
...
         "part3": "If a sensor in your area has been down for a long time, feel free to contact us and we’ll communicate with the sensor host to see if we can resolve the issue."
     },
+    "noSensorInArea": {
+        "heading": "No Sensors In Your Area?",
+        "body": "Some areas of South Gate are better represented on our map than others. Each sensor on the map is physically located somewhere, usually in the backyard of a generous South Gate resident. If there is an area of the city that you would like to see on the map, contact us and let us know! It is especially helpful if you know someone in the area who would be willing to host a sensor in their yard."
```
Comment (CarsonHerness) [inline public/locales/en/about.json:37]: Maybe add that a sensor needs a WiFi connection and an outlet

File: public/locales/en/about.json [hunk 2]
```diff
@@ -32,6 +32,10 @@
...
         "part3": "If a sensor in your area has been down for a long time, feel free to contact us and we’ll communicate with the sensor host to see if we can resolve the issue."
     },
+    "noSensorInArea": {
+        "heading": "No Sensors In Your Area?",
+        "body": "Some areas of South Gate are better represented on our map than others. Each sensor on the map is physically located somewhere, usually in the backyard of a generous South Gate resident. If there is an area of the city that you would like to see on the map, contact us and let us know! It is especially helpful if you know someone in the area who would be willing to host a sensor in their yard (the sensor requires WiFi and a power outlet)."
```
Comment (CarsonHerness) [inline public/locales/en/about.json:37]: Maybe add the contact info here or direct them to the Get Involved section, but I think a quick repeat of the contact info would be fine too

**Review by CarsonHerness — APPROVED** [review]
Looks good!

**Pull request merged**